mlflow/langchain/_langchain_autolog.py

```diff
@@ -213,8 +215,21 @@ def __init__(self, *args, **kwargs):
         )
 
     run_id = getattr(self, "run_id", None)
-    if (active_run := mlflow.active_run()) and run_id is None:
-        run_id = active_run.info.run_id
+    active_run = mlflow.active_run()
+    if run_id is None:
+        # only log the tags once
+        extra_tags = get_autologging_config(mlflow.langchain.FLAVOR_NAME, "extra_tags", None)
+        # include run context tags
+        resolved_tags = context_registry.resolve_tags(extra_tags)
+        tags = _resolve_extra_tags(mlflow.langchain.FLAVOR_NAME, resolved_tags)
+        if active_run:
```

Comment:
```suggestion
        if active_run := mlflow.active_run():
```

Can we call `active_run` here?

Reply:
No because it's used at line 314